.github/workflows/on_pr.yml

```
@@ -0,0 +1,46 @@
+name: On PR
+
+on: 
+  pull_request
```

Actually, this can be
```suggestion
  pull_request_target
```

It's a bit safer that way.

---

Sounds good. I will do it in a follwup PR

---

Sounds good - will do that in a followup PR or else it does not run with this PR.